bittide-extra/src/Clash/Reset/Extra.hs

```diff
@@ -0,0 +1,56 @@
+-- SPDX-FileCopyrightText: 2022-2023 Google LLC
+--
+-- SPDX-License-Identifier: Apache-2.0
+
+module Clash.Reset.Extra where
```

Comment:
Can we remove this module with the new clash hash?

Reply:
Yes! I've added comment:

```haskell
-- TODO: Remove these functions after merging clash-compiler#2539
```